README.md

```diff
@@ -25,7 +25,7 @@ If you are using Windows and python was installed using the [python installer](h
 
 From within the python environment described in the [Environment](#environment) section, run the command:
 
-`pip install git+https://github.com/IM-USAF/opal-release-downloader.git`
+`pip install git+https://github.com/309thEDDGE/opal-release-downloader.git`
 
```

Comment:
Maybe explain at a high level what this command will do.

Installs dependencies, and places the executables download_opal_artifacts, ... etc and places them in your path. I found myself looking around for the download_opal_artifacts script, I didn't realize it was in my path right off the bat.